### DISCUSSION TITLE
Fans still spin in quiet mode

### DISCUSSION THREAD

**Discussion by pinakiitkgp** [BartoszCichecki/LenovoLegionToolkit#810] [Q&A] [answered]
Fans still spinning in quiet mode despite laptop is completely cool and no intensive task is running except a video playing on youtube opened in edge browser.

**Comment by BartoszCichecki** [comment]
And what am I supposed to do with this information?

**Reply by pinakiitkgp** [reply]
Nothing if you just copy the vantage settings.. But if you have access to change settings on quiet mode, it will be great if a better fan curve can be set. 

Wondering if custom mode can be enabled on battery? Thanks!

**Reply by BartoszCichecki** [reply, marked as answer]
Fan curves are not modifiable in Quiet, Balance and Performance modes.

**Reply by pinakiitkgp** [reply]
Thanks! Any plan to implement custom mode on battery?

**Reply by BartoszCichecki** [reply]
Check the readme (FAQ specifically).
